qa-pipelines/tasks/lib/cf-deploy-upgrade-common.sh

```diff
@@ -110,24 +122,25 @@ get_internal_ca_cert() (
       | base64 -d
 )
 
-set_psp() {
-    HELM_PARAMS+=(--set "kube.psp.nonprivileged=suse.cap.psp.nonprivileged")
-    HELM_PARAMS+=(--set "kube.psp.privileged=suse.cap.psp.privileged")
-}
-
 # Helm parameters common to UAA and SCF, for helm install and upgrades
 set_helm_params() {
     HELM_PARAMS=(--set "env.DOMAIN=${DOMAIN}"
                  --set "secrets.UAA_ADMIN_CLIENT_SECRET=${UAA_ADMIN_CLIENT_SECRET}"
-                 --set "enable.autoscaler=true")
+                 --set "enable.autoscaler=true"
+                 --set "kube.storage_class.persistent=${STORAGECLASS}")
```

Comment:
unnecessary if storageclass name is always `persistent`

Reply:
not ture.